filters/auth/authclient.go

```diff
@@ -93,6 +93,9 @@ func (ac *authClient) getTokenintrospect(token string, ctx filters.FilterContext
 
 	if rsp.StatusCode != 200 {
 		io.Copy(io.Discard, rsp.Body)
+		if rsp.StatusCode != 403 && rsp.StatusCode != 401 {
+			return nil, fmt.Errorf("token introspection failed with status code: %d", rsp.StatusCode)
```

Comment:
Lets do just "failed with status code: %d" - the caller knows and logs what it calls (token introspection or tokeninfo)